### PR TITLE
Don't fetch widget data when it is collapsed

### PR DESCRIPTION
**Pull request by xavijam**
Also, when collapsed is disabled again, fetch the widget data if the ```url``` or the ```boundingBox``` attribute has changed since last time. And collapsed state will be only 22px of height.

Idea by @saleiva :clap:.

collapsed-widgets

### CODE REVIEW

**Comment by saleiva** [comment]
22px doesnt seem correct. We can review tomorrow ;)

**Comment by xavijam** [comment]
:joy: I hate you!

**Comment by saleiva** [comment]
:*

**Comment by xavijam** [comment]
@javierarce could you take a look please? Just in case I'm missing something from histograms?

**Comment by javierarce** [comment]
:+1: they work fine :)

File: src/geo/ui/widgets/category/model.js [hunk 1]
```diff
@@ -86,6 +86,19 @@ module.exports = WidgetModel.extend({
       });
     }, this);
 
+    this.bind('change:collapsed', function(mdl, isCollapsed) {
```
Comment (alonsogarciapablo) [inline src/geo/ui/widgets/category/model.js]: This is also duplicated [here](https://github.com/CartoDB/cartodb.js/pull/883/files#diff-214e2a6e2deb1d445b62d408f340b33eR55). Do we need it here? Just double checking...

Comment (xavijam) [inline src/geo/ui/widgets/category/model.js]: Function _onChangeBinds is being overriden in the category model.

**Pull request merged**